src/components/Footer.js

```diff
@@ -1,19 +1,24 @@
 import React from 'react'
 
 const Footer = () => (
-  <div className='footer'>
-    <hr />
+  <div>
     <div className='formulas-info'>
-      Sposób obliczania punktów znajdziesz na stronie:&nbsp;&nbsp;
-      <a target='_blank'
-        href='http://www.agh.edu.pl/rekrutacja/studia-pierwszego-stopnia/rekrutacja-20152016/nowa-matura/'
-        alt='Link do strony AGH ze sposobem obliczania punktów'>
-        http://www.agh.edu.pl/rekrutacja/studia-pierwszego-stopnia/rekrutacja-20152016/nowa-matura/
-      </a>
+      <div className='formulas-info-content'>
+        Sposób obliczania punktów znajdziesz na stronie:&nbsp;&nbsp;
+        <a target='_blank'
+          href='http://www.agh.edu.pl/rekrutacja/studia-pierwszego-stopnia/rekrutacja-20152016/nowa-matura/'
+          alt='Link do strony AGH ze sposobem obliczania punktów'>
+          http://www.agh.edu.pl/rekrutacja/studia-pierwszego-stopnia/rekrutacja-20152016/nowa-matura/
```

Comment:
My bad, I should have moved the closing sign to the next line so it would be less misleading.